crates/metaslang/graph_builder/src/graph.rs

```diff
@@ -35,7 +35,7 @@ pub struct Graph<KT: KindTypes> {
     graph_nodes: Vec<GraphNode>,
 }
 
-pub(crate) type SyntaxNodeID = u32;
+pub(crate) type SyntaxNodeID = usize;
```

Comment:
should this just use `metaslang_cst::nodes::NodeID` instead?

Reply:
Good point!